burrow/Cargo.toml

```diff
@@ -39,6 +39,8 @@ futures = "0.3.28"
 once_cell = "1.19"
 console-subscriber = { version = "0.2.0" , optional = true }
 console = "0.15.8"
+axum = "0.7.4"
+reqwest = { version = "0.11.24", features = ["json"] }
```

Comment:
I think we should consider the `rustls-tls-webpki-roots` feature to fix our Linux build error and to preserve consistency across platforms

Reply:
Done!